tasks/sync-granule/schemas/config.json

```
@@ -148,6 +148,10 @@
       "type": "boolean",
       "description": "If true, checksum files are also synced. Default: false.",
       "default": false
+    },
+    "workflowStartTime": {
+      "type": "integer",
+      "description": "Specifies the start time for the current workflow and will be used as the createdAt time for granules output"
```

```suggestion
      "description": "Specifies the start time for the current workflow (as a timestamp) and will be used as the createdAt time for granules output"
```

Worth clarifying that it must be a timestamp (as seconds from the epoch) in order for the comparison with Date.now() to work